.travis.yml

```diff
@@ -26,6 +26,9 @@ matrix:
     # Test against dev versions
     - php: 5.6
```

Comment:
Should we change this to PHP 7? Or add an explicit job for Symfony 4?

Reply:
I agree about changing this to PHP 7.1 (dev deps should be tested against a recent PHP version).

And we should add a LTS job for 3.4 (similar to what is done for 2.8, which should not be using dev deps anymore btw)